bot.py

```diff
@@ -102,9 +124,13 @@ async def on_message(message):
             await message.channel.send(f"Daddy Roth says: don't forget to {think_str}")
         if message.channel.id in target_channel_id and re.search(r"\bbot\b", message.content.lower()):
             result = classifier(message.content.lower())[0]
-            if result["label"] == "NEGATIVE":
+            #add in confidence count for positive and negative comments 
+            if result["label"] == "NEGATIVE" and result["score'] >= 0.75:
```

Comment:
```suggestion
            if result["label"] == "NEGATIVE" and result["score"] >= 0.75:
```
should use a double quote consistently

Reply:
yes sir